pulsar-io/debezium/oracle/src/main/resources/debezium-oracle-source-config.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+tenant: "public"
+namespace: "default"
+name: "debezium-oracle-source"
+topicName: "debezium-oracle-topic"
+archive: "connectors/pulsar-io-debezium-oracle-2.9.0-SNAPSHOT.nar"
```

Comment:
we are hard coding the version here, I am not sure that this version will be changed while doing the release

In the PostGres connector we do not have this line, probably it is to be removed

Reply:
We do have that in multiple connectors, versions are obsolete, not sure why we have that
```
$ grep -rI "archive:" . | grep -v /target/ 
./debezium/oracle/src/main/resources/debezium-oracle-source-config.yaml:archive: "connectors/pulsar-io-debezium-oracle-2.9.0-SNAPSHOT.nar"
./debezium/mongodb/src/main/resources/debezium-mongodb-source-config.yaml:archive: "connectors/pulsar-io-debezium-mongodb-2.4.0-SNAPSHOT.nar"
./debezium/postgres/src/main/resources/debezium-postgres-source-config.yaml:archive: "connectors/pulsar-io-debezium-postgres-2.4.0-SNAPSHOT.nar"
./debezium/mysql/src/main/resources/debezium-mysql-source-config.yaml:archive: "connectors/pulsar-io-debezium-mysql-2.4.0-SNAPSHOT.nar"
```

Similarly, the config.yaml files have example configuration. 
I don't mind removing that (as a separate change) if we agree that's the right thing to do.
Or the release procedure/script needs update to patch the version in these configs.